docs/output.md

```diff
@@ -1,43 +1,526 @@
-
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
-## Pipeline overview
-
-The pipeline is built using [Nextflow](https://www.nextflow.io/)
-and processes data using the following steps:
-
-* [FastQC](#fastqc) - read quality control
-* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
-
-## FastQC
-
-[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
-
-For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
-
-> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality. To see how your reads look after trimming, look at the FastQC reports in the `trim_galore` directory.
-
-**Output directory: `results/fastqc`**
-
-* `sample_fastqc.html`
-  * FastQC report, containing quality metrics for your untrimmed raw fastq files
-* `zips/sample_fastqc.zip`
-  * zip file containing the FastQC report, tab-delimited data file and plot images
-
-## MultiQC
-
-[MultiQC](http://multiqc.info) is a visualisation tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available in within the report data directory.
-
-The pipeline has special steps which allow the software versions used to be reported in the MultiQC output for future traceability.
-
-**Output directory: `results/multiqc`**
-
-* `Project_multiqc_report.html`
-  * MultiQC report - a standalone HTML file that can be viewed in your web browser
-* `Project_multiqc_data/`
-  * Directory containing parsed statistics from the different tools used in the pipeline
-
-For more information about how to use MultiQC reports, see [http://multiqc.info](http://multiqc.info)
+# ![nf-core/viralrecon](images/nf-core-viralrecon_logo.png)
+
+This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline. Please click [here](https://raw.githack.com/nf-core/viralrecon/master/docs/html/multiqc_report.html) to see an example MultiQC report generated using the parameters defined in [this](../conf/test_full.config) file to run the pipeline on [samples](https://zenodo.org/record/3735111) which were prepared from the [ncov-2019 ARTIC Network V1 amplicon set](https://artic.network/ncov-2019) and sequenced on the Illumina MiSeq platform in 301bp paired-end format.

```

Comment:
You mean in the pipeline template? Problem is that its something that has to be generated and updated specifically for that pipeline. Along with a description of what it contains. Could maybe add it as a comment but it could be something that can be easily missed unless you remember to update it 😅

Reply:
I have added it to the main `README.md` too and in the section explaining the `MultiQC` 
 outputs. Agree its a nice way to showcase the metrics reported by the pipeline though 👍